content/documentation/PinePhone_Pro/Software/Multi-distribution_image.adoc

```diff
@@ -4,16 +4,16 @@ draft: false
 menu:
   docs:
     title:
-    parent: "PinePhone_Pro/Various"
-    identifier: "PinePhone_Pro/Various/Multi-distribution_image"
+    parent: "PinePhone_Pro/Variuos"
+    identifier: "PinePhone_Pro/Variuos/Multi-distribution_image"
```

Comment:
Typo. Also in that case you need to `git mv` it back to the right section.

To minimize commit clutter, consider `git reset`ing your branch back to what's in upstream and making your changes from there. Note that this would involve a force-push.